packages/ariakit/src/collection/__examples__/collection/test.tsx

```diff
@@ -0,0 +1,11 @@
+import { getByText, render } from "ariakit-test-utils";
+import Example from ".";
+
+test("render correctly", async () => {
+  render(<Example />);
+  expect(getByText(/apple/i)).toBeInTheDocument();
+  expect(getByText(/grape/i)).toBeInTheDocument();
+  expect(getByText(/orange/i)).toBeInTheDocument();
+  expect(getByText(/strawberry/i)).toBeInTheDocument();
+  expect(getByText(/watermelon/i)).toBeInTheDocument();
+});
```

Comment:
Now we can check if the state is working properly:
```suggestion
test("show items count", async () => {
  render(<Example />);
  expect(getByText("Items count: 3")).toBeInTheDocument();
});
```

Reply:
Updated in https://github.com/reakit/reakit/pull/977/commits/a379ef1778a11c9bfa5c0a60607cb3996e95046c